rotkehlchen/chain/ethereum/modules/balancer/utils.py

```diff
@@ -142,15 +287,15 @@ def deserialize_swap(raw_swap: Dict[str, Any]) -> AMMSwap:
         raise DeserializationError(f'Missing key: {str(e)}.') from e
 
     if amount0_in == ZERO:
-        # Prevent a DivisionByZero when creating the trade
-        raise DeserializationError('TokenAmountIn balance is zero')
+        # Prevent a ZeroDivisionError when creating the trade
```

Comment:
Why rename this? This called a "DivisionByZero" error. Not a ZeroDivision error

Reply:
the python exception class name is `ZeroDivisionError`. I will just write "prevent a division by zero..."